examples/use-cases/push_events_to_syslog/README.md

```diff
@@ -0,0 +1,41 @@
+# Push Events to Syslog Daemon
+The push events to syslog daemon example provides the base for users to view a history of [events](https://developer.automox.com/openapi/axconsole/operation/getEvents/) happening on their Automox instance in their preferred syslog environments.
```

Comment:
Maybe we call out a big more that modifications for environments are expected.

```suggestion
The push events to syslog daemon example provides the base for users to view a history of [events](https://developer.automox.com/openapi/axconsole/operation/getEvents/) happening on their Automox instance in their preferred syslog environments. This use case is intended to provide an example of how to use the SDK with Automox events, track them, and send off to a third-party. Due to the varying ways to send data to a SIEM this script will likely need modified to meet environment and/or data needs.
```

Reply:
This is a good idea, I'll commit it.